src/cromshell/utilities/VariableSetup.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+
+def get_submission_file(config_dir):
+    submission_file = "{}/all.workflow.database.tsv".format(config_dir)
+    if not Path.exists(Path(submission_file)):
+        Path.touch(Path(submission_file))
+        submission_header = 'DATE{0}CROMWELL_SERVER{0}RUN_ID{0}WDL_NAME{0}STATUS{0}ALIAS'.format('\t')
+        with Path(submission_file).open() as f:
+            f.write(submission_header)
+    return submission_file
+
+
+def get_config_dir():
+    config_path = str(Path.home()) + "/.cromshell"
+    Path.mkdir(Path(config_path), exist_ok=True)
+    return config_path
+
+
+def get_cromwell_server(config_dir):
+    cromwell_server_path = "{}/cromwell_server.config".format(config_dir)
+    if Path.exists(Path(cromwell_server_path)):
+        with Path(cromwell_server_path).open() as f:
+            cromwell_server = f.readline().strip()
+        return cromwell_server
+    else:
+        print("Could not find cromwell_server.config")
+        print("Creating file {}".format(cromwell_server_path))
+        Path.touch(Path(cromwell_server_path))
+        print("Please add a Cromwell server URL to file {}".format(cromwell_server_path))
+        quit()
+
+
+class Cromwell(object):
```

Comment:
`object` shouldn't be necessary

Reply:
And I'd suggest giving the class a more appropriate name, as `Cromwell` is too general.

Reply:
done, removed "object", renamed class to CromshellConfig